Userland/Shell/Shell.cpp

```diff
@@ -1194,6 +1194,27 @@ String Shell::unescape_token(const String& token)
     return builder.build();
 }
 
+String Shell::find_in_path(String program_name)
```

Comment:
`String const&`?

Reply:
Actually, I think I'll make it take a `const StringView&` (and not a `StringView const&` - boo)

Reply:
Done : )